### PR TITLE
fix(docs): updates the Tyk Cloud trial quick start guide

### PR DESCRIPTION
**Pull request by excieve**
### Preview Link
https://deploy-preview-5533--tyk-docs.netlify.app/docs/nightly/deployment-and-operations/tyk-cloud-platform/quick-start/
<br>

### Description
Tyk Cloud onboarding into a free trial has changed. This change updates the trial quick start guide accordingly. It also updates the Tyk Dashboard screenshots to ones made with a recent version.
<br>

## Checklist

- [x] I have added a **preview link** to the PR description.
- [ ] I have reviewed the suggestions made by our AI (PR Agent) and updated them accordingly (spelling errors, rephrasing, etc.)
- [x] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [x] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [x] Make sure you have started *your change* off *our latest `master`*.
- [x] I **labeled** the PR

### CODE REVIEW

**Review by sophieriches — COMMENTED** [review]
Looks great. I think it would be worth adding a small note section at the top of the page with information stating that 'The Tyk Cloud trial is limited to 48 hours, after which time your data will be deleted. The Tyk Cloud free trial does not include access to Hybrid deployments or the Developer Portal, to try out these capabilities, get in touch for a guided evaluation with our team'

**Comment by excieve** [comment]
> Looks great. I think it would be worth adding a small note section at the top of the page with information stating that 'The Tyk Cloud trial is limited to 48 hours, after which time your data will be deleted. The Tyk Cloud free trial does not include access to Hybrid deployments or the Developer Portal, to try out these capabilities, get in touch for a guided evaluation with our team'

Thanks @sophieriches! It's added now.

File: tyk-docs/content/deployment-and-operations/tyk-cloud-platform/quick-start.md [hunk 1]
```diff
@@ -9,95 +9,110 @@ menu:
...
-* Finally, click on the button "CONFIGURE API".
 
-{{< img src="/img/cloud/create-api-tyk-cloud.png" alt="Accessing Creating an API" width="500px" >}}
+* Finally, click on the button "CONFIGURE API".
+{{< img src="/img/cloud/create-api-tyk-cloud.png" alt="Designing an API" >}}
 
 * Select to which gateway you want to deploy this API, select the "edge" tags to deploy to the cloud data plane.
 
-{{< img src="/img/cloud/tyk-cloud-select-cloud-gateway.png" alt="Accessing Creating an API" width="500px" >}}
+{{< img src="/img/cloud/tyk-cloud-select-cloud-gateway.png" alt="Selecting Cloud Data Planes for an API" >}}
```
Comment (letzya) [inline tyk-docs/content/deployment-and-operations/tyk-cloud-platform/quick-start.md]: @sharadregoti @excieve can we please leave the current width of 500.  Without it, it's too big. The best would be to get a new screenshot with the new UI, but if not, just leave the width on the images that do close-ups, so they are not disproportional to the rest of the page

File: tyk-docs/content/deployment-and-operations/tyk-cloud-platform/quick-start.md [hunk 2]
```diff
@@ -9,95 +9,110 @@ menu:
...
 * Go to the "Configurations" section and give your key an alias (e.g., 'platform_team').
 * Click on the button "Create Key".
 
-{{< img src="/img/cloud/tyk-cloud-key-configurations.png" alt="Key section" width="500px" >}}
+{{< img src="/img/cloud/tyk-cloud-key-configurations.png" alt="Key configuration" >}}
 
 Congratulations! Your key has now been created!
 
-{{< img src="/img/cloud/tyk-cloud-copy-key-url.png" alt="Key section" width="500px" >}}
+{{< img src="/img/cloud/tyk-cloud-copy-key-url.png" alt="Copy Key ID" >}}
```
Comment (letzya) [inline tyk-docs/content/deployment-and-operations/tyk-cloud-platform/quick-start.md:116]: same here https://github.com/TykTechnologies/tyk-docs/pull/5533/files#r1793888695

**Review by letzya — DISMISSED** [review]
Asked to put back the size limit of a few images https://github.com/TykTechnologies/tyk-docs/pull/5533/files#r1793888695

**Comment by excieve** [comment]
@letzya Done, please take a look again,

**Comment by letzya** [comment]
@excieve unfortunately the screenshots were on a wide screen not on a regular mac and that makes the font looks too small to see https://deploy-preview-5533--tyk-docs.netlify.app/docs/nightly/deployment-and-operations/tyk-cloud-platform/quick-start/

**Comment by excieve** [comment]
@letzya I don't own a Mac. And I suspect the resolution could differ in different models. Is there any guidance on what the font size and screen resolution should be on the screenshots in our docs?

**Review by sharadregoti — APPROVED** [review]

**Comment by sharadregoti** [comment]
I'll update the images with wide screen-shots in another PR.

**Pull request merged**